web-configs/nginx/nginx-no-ssl.conf

```diff
@@ -0,0 +1,59 @@
+# Define the location for where cobalt web runs. Change the port to fit your needs.
+upstream cobaltweb {
+    server 127.0.0.1:9000;
+    keepalive 64;
+}
+
+# Define the location for where cobalt API runs. Change the port to fit your needs.
+upstream cobaltapi {
+    server 127.0.0.1:9001;
+    keepalive 64;
+}
+
+# Server block definition for cobalt web.
+server {
+    # Listen for for requests to the domain on port 80.
+    listen 80;
+    listen [::]:80;
+    server_name co.wukko.sh;
+	
+    # Set loggging.
+    access_log /var/log/nginx/co.wukko.sh.access.log;
+    error_log /var/log/nginx/co.wukko.sh.error.log;
+
+    # Pass all requests to cobalt web, as defined above.
+    location / {
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Forwarded-Server $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```

Comment:
also pointless unless cobalt starts using `trust proxy` (which it should, btw)

\+ this requires additional config for it to have a meaningful value if the instance owner is using cloudflare, and that should definitely be documented somewhere, since cobalt seems to expect public instance operators to place it behind cloudflare